Knoema.Client/Data/FullDimensionRequest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Knoema.Data
+{
+	public class FullDimensionRequest
+	{
+		public List<DimensionRequestItem> DimensionRequest { get; set; }
+		public int Calendar { get; set; }
+		public DateTime? StartDate { get; set; }
+		public DateTime? EndDate { get; set; }
+		public int? DateColumn { get; set; }
+		public List<string> Frequency { get; set; }
+		public IEnumerable<TimeseriesAttributeModel> TimeseriesAttributes { get; set; }
+	}
+
+	public class DimensionRequestItem
+	{
+		public string DimensionId { get; set; }
+		public List<int> Members { get; set; }
```

Comment:
Maybe need to add ,members creation to ctor

Reply:
fixed, see next commit